scripts/lint-bash.sh

```diff
@@ -26,28 +26,6 @@ mapfile -t check_files < <(
     uniq -u
 )
 
-# Handle bats tests
-bats_tmp="$(mktemp -d lint-bash-XXXXXX)"
-function cleanup() {
-  rm -rf "${bats_tmp}"
-}
-trap cleanup EXIT
-
-readonly linter=koalaman/shellcheck:v0.6.0
-
-if ! docker image inspect "$linter" &>/dev/null; then
-  docker pull "$linter"
-fi
-
-cmd=(docker run -v "$(pwd):/mnt")
-if [ -t 1 ]; then
-  cmd+=(--tty)
-fi
-cmd+=(
-  --rm
-  "$linter" "${check_files[@]}"
-)
-
 echo "Linting scripts..."
-"${cmd[@]}"
+shellcheck "${check_files[@]}"
```

Comment:
shellcheck is installed in the buildenv?

Reply:
oh, i see you added it. but you didn't set the version. Should we be worried about the version changing randomly?

Reply:
it is with this change

Reply:
has anything changed since 0.6.0? what's the new version?

Reply:
The current version this is installing is 0.7.1. shellcheck complained about the wordsplitting in generate-clientset which I fixed in this change, but that was the only issue.

https://packages.debian.org/bullseye/shellcheck